libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1214,8 +1214,6 @@ bool AP_InertialSensor::calibrate_trim(float &trim_roll, float &trim_pitch)
         return false;
     }
 
-    _calibrating_accel = true;
```

Comment:
Should we replace this with `_calibrating_trim` instead?  The externally-visible `::calibrating()` will be incorrect after this.

Reply:
that would give the wrong result in GCS.cpp where calibrating is used